src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -450,7 +438,7 @@ std::wstring CascadiaSettings::ExpandEnvironmentVariableString(std::wstring_view
     do
     {
         result.resize(requiredSize);
-        requiredSize = ::ExpandEnvironmentStringsW(source.data(), result.data(), gsl::narrow<DWORD>(result.size()));
+        requiredSize = ::ExpandEnvironmentStringsW(source.data(), result.data(), static_cast<DWORD>(result.size()));

```

Comment:
Why was this changed?

Reply:
Sorry, that's an artifact of my trying to learn git.  I didn't change this, something must have happened when I was bringing in the updated codebase between my revisions.

I think I need to scratch this branch and start anew, something went wrong when syncing this time and now I can't compile anymore.